src/main/java/nl/hsac/fitnesse/fixture/slim/XmlFixture.java

```diff
@@ -90,6 +90,15 @@ public String createFileFromBase64ContentOf(String baseName, String xPath) {
         }
     }
 
+    /**
+     * Validat the loaded xml against a schema in file xsdFileName
+     * @param xsdFileName filename of the xsd to use
+     * @return true if the xml validates against the schema. Throws a descriptive exception otherwise
+     */
+    public boolean validateAgainst(String xsdFileName) {
+        return new XMLValidator().validateAgainst(content, xsdFileName);
```

Comment:
Please extract the content from the xsd using FileFixture here, and not in the util class.